pkgs/os-specific/linux/ply/default.nix

```
@@ -34,9 +32,10 @@ in stdenv.mkDerivation {
   '';
 
   meta = with lib; {
-    description = "dynamic Tracing in Linux";
+    description = "Dynamic tracing in Linux";
     homepage = "https://wkz.github.io/ply/";
     license = [ licenses.gpl2 ];
     maintainers = with maintainers; [ mic92 mbbx6spp ];
+    broken = lib.versionAtLeast kernel.version "4.0";
```

Should it not be the other way around?

```suggestion
    broken = lib.versionOlder kernel.version "4.0";
```